### PR TITLE
Open the store from the machine view onboarding

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #2645. Fixes: #2644.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- open the machine view
- click the "Add applications to get started" text
- the store should open

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7046/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7047/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7048/

File: jujugui/static/gui/src/app/components/machine-view/test-machine-view.js [hunk 1]
```diff
@@ -195,29 +199,67 @@ describe('MachineView', function() {
...
       </div>);
-    assert.deepEqual(
-      output.props.children.props.children[0].props.children[1], expected);
+    expect(
+      output.props.children.props.children[0].props.children[1]).toEqualJSX(
+      expected);
+  });
+
+  it('can open the store from the onboarding', function() {
+    var units = {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/machine-view/test-machine-view.js]: const

File: jujugui/static/gui/src/app/components/machine-view/test-machine-view.js [hunk 2]
```diff
@@ -195,29 +199,67 @@ describe('MachineView', function() {
...
+      filterByMachine: sinon.stub().returns([])
+    };
+    var services = {
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/machine-view/test-machine-view.js]: const

**Comment by frankban** [comment]
QA is ok!

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7062/

**Comment by huwshimi** [comment]
Thanks @hatched @frankban :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**